src/libraries/System.Data.Common/src/System/Data/ProviderBase/SchemaMapping.cs

```diff
@@ -56,6 +56,13 @@ internal sealed class SchemaMapping
 
         private readonly LoadOption _loadOption;
 
+        [RequiresUnreferencedCode("dataReader schema table rows DataTypes type cannot be statically analyzed.")]
+        internal SchemaMapping(DataAdapter adapter, DataSet? dataset, DataTable? datatable, DataReaderContainer dataReader, bool keyInfo,
+                                    SchemaType schemaType, string? sourceTableName, bool gettingData)
+            : this(adapter, dataset, datatable, dataReader, keyInfo, schemaType, sourceTableName, gettingData, null, null)
+        { }
+
+        [RequiresUnreferencedCode("chapterValue and dataReader schema table rows DataTypes type cannot be statically analyzed.")]
```

Comment:
Do we still need to split this internal ctor now that they are both marked `[RUC]`?

Reply:
no strong opinion either way, will remove